include/qvs.hpp

```diff
@@ -0,0 +1,175 @@
+#ifndef QVS_HPP
+#define QVS_HPP
```

Comment:
I am getting a compilation warning from line 51, please fix it

Reply:
Could I get a photo of these warnings? I just deleted build directory and re-built the code and I am not getting this warning.

Reply:
I get this warning: ```src/../include/qvs.hpp:48:16: warning: private field 'rs' is not used [-Wunused-private-field]``` It should be enough to unused rs variable